openpower/configs/witherspoon_defconfig

```diff
@@ -42,7 +42,7 @@ BR2_ROOTFS_DEVICE_TABLE="../openpower/device_table.txt"
 BR2_ROOTFS_POST_BUILD_SCRIPT="../openpower/scripts/fixup-target-var"
 BR2_LINUX_KERNEL=y
 BR2_LINUX_KERNEL_CUSTOM_VERSION=y
-BR2_LINUX_KERNEL_CUSTOM_VERSION_VALUE="4.4.18"
+BR2_LINUX_KERNEL_CUSTOM_VERSION_VALUE="4.4.19"
```

Comment:
Although in issue https://github.com/open-power/op-build/issues/611 @mikey states that we should move up to 4.8, currently 4.4.18 does not compile as https://github.com/open-power/op-build/blob/master-next/openpower/linux/0009-Release-4.4.19-openpower1.patch does not apply nicely.